src/runtime/bytecode/AstGen.zig

```diff
@@ -146,8 +147,9 @@ fn generateBlock(self: *AstGen, executable: *Executable, block: *Block, block_no
 
     const block_index = try self.generateSlotsAndCodeCommon(executable, block, block_node.slots, block_node.statements.value.statements);
     const block_location = self.allocateRegister();
+    const slot_count: u32 = @intCast(block_node.slots.len);
     try block.addInstruction(executable.allocator, .CreateBlock, block_location, .{
-        .slot_count = @intCast(u32, block_node.slots.len),
+        .slot_count = slot_count,
```

Comment:
I remember having a problem with these when I attempted to do an `intCast` update myself. Do you know whether `@intCast` still fails to infer the type for generic functions here? I'd rather have this do a `@intCast` directly into the struct. (not a blocker)

Reply:
Yes, it still fails. That's why I moved it to a separate line.